public/js/home.js

```diff
@@ -1130,6 +1132,26 @@ function openTab(evt, tab_name) {
     recentAttendance.redraw();
 }
 
+function openSideNav() {
+    let sidenav = document.getElementById("sidenav");
+    if (sidenav.clientWidth == 270) sidenav.style.width = "0px";
+    else sidenav.style.width = "270px";
```

Comment:
Triple equals and use ternary operator (instead of if/else without braces)
```suggestion
    sidenav.style.width = sidenav.clientWidth === 270 ? "0px" : "270px";
```